lib/engine/step/g_1867/token.rb

```diff
@@ -7,6 +7,18 @@ module Engine
   module Step
     module G1867
       class Token < Token
+        def auto_action(entity)
+          Engine::Action::Pass.new(entity) unless can_really_place_token?(entity)
```

Comment:
One slight flaw with how this is done is it'll log a pass rather than skipping

A solution would be to store if the Pass was a "skip" and tweak the debug appropriately